### PR TITLE
Updated to work with GTNH 2.5.0, bumped version, updated dependencies…

### PR DESCRIPTION
**Pull request by makonew**
…, fixed com.github.dcysteine.nesql.exporter.plugin.gregtech.util.RecipeMap to work with GT5 after refactor of Recipe maps

### CODE REVIEW

File: src/main/java/com/github/dcysteine/nesql/sql/quest/QuestLine.java [hunk 1]
```diff
@@ -3,11 +3,7 @@
 import com.github.dcysteine.nesql.sql.Identifiable;
 import com.github.dcysteine.nesql.sql.Metadata;
 import com.github.dcysteine.nesql.sql.base.item.Item;
-import jakarta.persistence.Column;
```
Comment (boubou19) [inline src/main/java/com/github/dcysteine/nesql/sql/quest/QuestLine.java:6]: no star imports, please configure your IDE to avoid this.

**Review by boubou19 — CHANGES_REQUESTED** [review]

**Comment by boubou19** [comment]
@Makonew-Dev please address the review and this will be merged.

**Comment by Dream-Master** [comment]
@Makonew-Dev any news

**Comment by D-Cysteine** [comment]
Sorry for the long delay on this. I've been working on updating the exporter to work with the latest version of the pack, so this PR should no longer be needed once I've finished that.

**Comment by Dream-Master** [comment]
@D-Cysteine we releasing 2.6.0 today soone more bump :)

**Pull request closed without merging**